ethers-core/Cargo.toml

```diff
@@ -14,6 +14,7 @@ rlp = { version = "0.5.0", default-features = false }
 ethabi = { version = "15.0", default-features = false, features = ["full-serde", "rlp"] }
 arrayvec = { version = "0.7.2", default-features = false }
 rlp-derive = { version = "0.1.0", default-features = false }
+cargo_metadata = "0.14.0"
```

Comment:
Should this be `optional = true` and enabled only if `macros` is enabled?

Reply:
Yeah that makes sense since it's only used by the macros crate for now